### ISSUE TITLE
Try enabling statusreconciler

### ISSUE THREAD

**Issue by bobcatfish** [tektoncd/plumbing#54]
# Expected Behavior

Our prow deployment should use all useful and available Prow tools.

# Actual Behavior

Since we installed Prow, a new component called `statusreconciler` was added (https://github.com/kubernetes/test-infra/pull/12258) - it looks like this has something to do with preventing PRs from becoming stuck? Would be good to understand what this actually does before turning it on.

# Additional Info

Example configuration is available at:

https://github.com/kubernetes/test-infra/blob/6a85e1e9718d1d43aaa4b756826b36e8d22eff71/prow/cluster/starter.yaml#L456-L504

**Comment by bobcatfish** [comment]
As PRs such as https://github.com/tektoncd/pipeline/pull/1161 hang with no status reported, I become increasingly concerned that this component is essential 😅 

![image](https://user-images.githubusercontent.com/432502/62487450-def03000-b78f-11e9-82e8-8c4785b9f03f.png)

**Comment by bobcatfish** [comment]
Nope, that was a totally different problem. Phew.

**Comment by bobcatfish** [comment]
From @cjwagner it sounds like this tool is for retrying statuses when a job (prow job?) config changes.

**Comment by cjwagner** [comment]
Yes, there are some docs with more detail here: https://github.com/kubernetes/test-infra/tree/master/prow/cmd/status-reconciler#status-reconciler
